wikiconv/README.md

```diff
@@ -0,0 +1,120 @@
+# WikiConv
+
+WikiConv is a set of corpora encompassing conversations happens on Wikipedia.
+The reconstruction process of this corpus has been published in EMNLP 2018
+[WikiConv: A Corpus of the Complete Conversational History of a Large Online
+Collaborative Community](TBA).
+The second half of [this Wikipedia reasearch
+showcase](https://www.mediawiki.org/wiki/Wikimedia_Research/Showcase#June_2018) presents the corpus and
+the methodology.
+
+WikiConv is a multi-language corpus, it currently includes:
+
+- [English (20180701 data dump)]()
```

Comment:
Missing links?

Reply:
Yes, I want to put the public link here (kaggle or figshare).